File: motor3d 2.0/objetoalgebraico.vb
Namespace Algebra
    Public MustInherit Class ObjetoAlgebraico
        Inherits ObjetoBase

        Public Shadows Event Modificado(ByRef Sender As ObjetoAlgebraico)

        Public Sub New()
            MyBase.New()
        End Sub

        Public Overrides Function ToString() As String
            Return MyBase.ToString()
        End Function
    End Class
End Namespace

File: motor3d 2.0.ConvertedToC#/ObjetoAlgebraico.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
using System.Linq;
using System.Xml.Linq;
namespace Motor3D.Algebra
{
	public abstract class ObjetoAlgebraico : ObjetoBase
	{

		public new event ModificadoEventHandler Modificado;
		public new delegate void ModificadoEventHandler(ref ObjetoAlgebraico Sender);

		public ObjetoAlgebraico() : base()
		{
		}

		public override string ToString()
		{
			return base.ToString();
		}
	}
}
